game.py

```diff
@@ -0,0 +1,116 @@
+import json, os
+from colorama import init, Fore, Style
+from random import choice, randint
+from pokemon import pokemon
+
+class game:
+    def __init__(self):
+        # Define Variables
+        self.clear_cmd = lambda: os.system('cls')
```

Comment:
`os.system('cls' if os.name=='nt' else 'clear')` recommended for cross platform compatability.